doc/mgmt/Management Framework.md

```diff
@@ -1081,6 +1092,118 @@ go run gnmi_set.go -replace  /openconfig-acl:acl/acl-sets/acl-set[name=MyACL4][t
+jwt_refresh = 1-jwt_valid           
+                                    ; The number of seconds before the JWT    ; token expires that you can refresh the  ; token. - defaults to 900 seconds.
+jwt_valid = 1-2^64        
+                                    ; Number of seconds the JWT tokens are    ; valid. - defaults to 3600          
+
+```
+
+###### 3.2.2.5.6 Transport options
+
+gNMI server supports only TCP transport with TLS enabled and listens on default port 8080.
+Server port can be changed through an entry in ConfigDB TELEMETRY table.
+
+By default a temporary self signed certificate is used as TLS server certificate.
+It can be overridden by specifiying a valid TLS private key and certificate file paths through the TELEMETRY table.
+
+###### 3.2.2.5.7 Translib linking
+
+gNMI server will statically link with Translib. For each gNMI request, the server
+invokes Translib API which then invokes appropriate App module to process the request.
+Below is the mapping of gRPC operations to Translib APIs:
+
+ gRPC Method | Translib API      | Request data  | Response data
+-------------|-------------------|---------------|---------------
+ GET         | translib.Get      | path          | status, payload
+ SET/UPDATE  | translib.Update   | path, payload | status
+ SET/REPLACE | translib.Replace  | path, payload | status
+ SET/DELETE  | translib.Delete   | path          | status
+ (ANY OC-RPC)| translib.Action   | RPC,payload   | status, payload
+ SUBSCRIBE   | translib.Subscribe| path          | status, payload
+ CAPABILITIES| translib.GetModels|               | models, versions, encodings
+ 
+
+More details about Translib APIs are in section [3.2.2.6](#3_2_2_6-Translib).
+
+
+###### 3.2.2.5.8 Encodings
+
+gNMI Currently only supports JSON_IETF type encoding in the same was as REST server since translib nativly uses this format.
+
+###### 3.2.2.5.9 Payload Validations
+
+gNMI server does not validate request payload for YANG defined RESTCONF APIs.
+Payload will be validated automatically in lower layers when it gets loaded
+into YGOT bindings.
+
+###### 3.2.2.5.10 Concurrency
+
+gNMI server will accept concurrent requests. Translib provides appropriate locking mechanism - parallel reads and sequential writes.
+
+###### 3.2.2.5.11 API Versioning
+
+gNMI server will allow clients to specify API version through a custom gNMI Extension field BundleVersion with extentsion id 700.
+
+    BundleVersion: 1.0.3
+
+Version text should be in **MAJOR.MINOR.PATCH** format. gNMI server will extract version text from
+the gNMI request protobuf object and pass it to the Translib API as **ClientVersion** argument.
+Section [3.2.2.6.4.2](#322642-version-checks) explains Translib version checking logic.
+
+Version checks are bypassed if Accept-Version header is not present in the request.
+
+For YANG defined RESTCONF APIs, the server's version can be discovered through the standard Capabilities gNMI RPC.
```

Comment:
I feel we should avoid using term "RESTCONF APIs" in gNMI section.
Also, I guess the capabilities PRC will return supported yang bundle version through an extension message. Please provide that information as well.

Reply:
Ok I added the proto definition and referenced the sonic.proto file.